roles/ceph-rgw/tasks/multisite/main.yml

```diff
@@ -19,6 +19,13 @@
   # Hard code this so we will skip the entire file instead of individual tasks (Default isn't Consistent)
   static: False
 
+- name: cleanup default zone
+  include: cleanup.yml
+  when:
+    - cleanup_default_zone is defined
```

Comment:
you need to add this var to `https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-rgw/defaults/main.yml`, `cleanup_default_zone: False`.

then need to run `generate_group_vars_sample.sh`.